devel/MacOSX.sdk/Portfile

```diff
@@ -381,6 +384,70 @@ subport MacOSX10.6.sdk {
     distname            Xcode_4.3_for_Lion
 }
 
+subport MacOSX10.5.sdk {
+#   SDK                   Size  Date        File                                                Notes
```

Comment:
I'm surprised there is a need for these old SDKs but I have no objection to adding them.

If you don't want to fill in all rows of this table (for all files that provide these SDKs) can you at least fill in the row for the file you're using?

Reply:
> I'm surprised there is a need for these old SDKs but I have no objection to adding them.

This is part of a foolish attempt to build rust-bootstrap for 10.5 on a newer machine.
MacOSX10.4.sdk and MacOSX10.3.sdk were just added for the sake of completeness.

> If you don't want to fill in all rows of this table (for all files that provide these SDKs) can you at least fill in the row for the file you're using?

I modified the pull request to include tables.